threescale_api/resources.py

```diff
@@ -1062,7 +1056,7 @@ def entity_id(self) -> int:
 
 
 class Application(DefaultResource):
-    def __init__(self, entity_name='name', **kwargs):
+    def __init__(self, entity_name='system_name', **kwargs):
```

Comment:
also this looks suspocious and dangerous, especially is the three changes arent aligned

Reply:
`Application` has not anything like `system_name` in `Service` which is unique index similar to `id`. But it seems that `name` is better because it is required parameter for creating of `Application`